scripts/check_sbml_metadata.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import sys
+
+import petab
+
+from lxml import etree
+from more_itertools import one
+
+from overview import get_reference_uris
+from _helpers import petab_yamls
+
+
+num_failures = 0
+
+for petab_problem_id, petab_yaml in petab_yamls.items():
+    petab_problem = petab.Problem.from_yaml(petab_yaml)
+
+    model_id = petab_problem.sbml_model.getId()
+    model_name = petab_problem.sbml_model.getName()
+    reference_uris = get_reference_uris(sbml_model=petab_problem.sbml_model)
+
+    errors = []
+
+    if model_id != petab_problem_id:
+        errors.append(
+            'Please change the model ID to match the problem ID.'
+        )
```

Comment:
This may become a problem once there are problems involving multiple models. Maybe then `.startswith()`? But okay for now, I think.

Reply:
Agreed, but there were some current model(s) that had long IDs prefixed by the problem ID... in the absence of a convention, we could wait until someone shares such a problem with multiple models, and get their opinion.